test/pbkdf2_tests.erl

```diff
@@ -25,9 +25,12 @@
 	{[sha, <<"passwordPASSWORDpassword">>, <<"saltSALTsaltSALTsaltSALTsaltSALTsalt">>, 4096, 25],
 			<<"3d2eec4fe41c849b80c8d83662c0e44a8b291a964cf2f07038">>},
 	{[sha, <<"pass\0word">>, <<"sa\0lt">>, 4096, 16],
-			<<"56fa6aa75548099dcc37d7f03425e0c3">>},
-	{[sha, <<"password">>, <<"salt">>, 16777216, 20],
-			<<"eefe3d61cd4da4e4e9945b3d6ba2158c2634e984">>}
+			<<"56fa6aa75548099dcc37d7f03425e0c3">>}
+
+	%% XXX: The following vector takes an impractical amount of time to test.
```

Comment:
Is this value impractical locally only? If that's the case, I'd prefer a tag/macro/profile for travis tests that'd still include these tests. Locally `rebar3 do eunit, ct` should suffice.

Reply:
I suspect it's impractical anywhere. If my modernish computer can't handle it in useful time (we're talking about _at least_ a few minutes - I didn't actually wait for completion), then the free CI potato servers won't, either.

Reply:
I'd keep it as is, and leave it up to future me to make an actual decision (as in either keep the test or remove it entirely.)